tests/causal_refuters/test_dummy_outcome_refuter.py

```diff
@@ -1,11 +1,13 @@
 import pytest
 import numpy as np
+import pdb
 from .base import TestRefuter
 
-def simple_linear_outcome_model(df):
+def simple_linear_outcome_model(X_train, output_train):

```

Comment:
why is output_train required here? I thought that the function argument was a df.
Also, I thought that the _true_causal_effect took only 1 parameter?

Reply:
Got it, then we need to update the docstring for __init__ above that talks about the causal_effect function and what parameters can it take.